sdk/loadtesting/arm-loadtesting/assets.json

```diff
@@ -2,5 +2,5 @@
   "AssetsRepo": "Azure/azure-sdk-assets",
   "AssetsRepoPrefixPath": "js",
   "TagPrefix": "js/loadtesting/arm-loadtesting",
-  "Tag": "js/loadtesting/arm-loadtesting_ae2944b5f7"
+  "Tag": "js/loadtesting/arm-loadtesting_350173f689"
```

Comment:
Could you help me understand what is the root cause for this failure?

Reply:
![image](https://github.com/Azure/azure-sdk-for-js/assets/98569699/c13998a3-fffa-4e96-ad9c-07d8484197a9)
lack of ```Content-Type``` in final get request